README.md

```diff
@@ -32,16 +32,34 @@ Related tools added to image:
 ./build.sh
 ```
 
+Build accepts the following flags:
+```
+  -h  --help      Show this message and exit
+  -t  --tag       Build with a specific docker tag
+  -x  --debug     Set the bash debug flag
+```
+
+You can also override the container build flags by separating them at the end of the command with `--`.  Example:
+```
+./build.sh -t local-dev -- --no-cache
+```
+
 ### Use it:
 ```
 ocm-container
 ```
 With launch options:
 ```
 OCM_CONTAINER_LAUNCH_OPTS="-v ~/work/myproject:/root/myproject" ocm-container
+--
+or
+--
+ocm-container -o "-v ~/work/myproject:/root/myproject" ocm-container
```

Comment:
Didn't see in the code usage of the -o flag.

I do think it's a gr8 addition but not sure where it's done

Reply:
Hmm, I explicitly remember adding that option (because I've used it locally before), but I must have missed it during a rebase or stashed it and forgot to get it back or something.  Good catch, I've added the option back in.